fv3core/.jenkins/actions/run_standalone.sh

```diff
@@ -111,13 +117,8 @@ echo "Benchmark directory:          ${BENCHMARK_DIR}"
 echo "Data directory:               ${DATA_DIR}"
 echo "Perf. artifact directory:     ${TIMING_DIR}"
 echo "Profile artifact directory:   ${PROFILE_DIR}"
-echo "Cache directory:              ${CACHE_DIR}"
```

Comment:
This was removed because CACHE_DIR is not used in this file, and because it gets printed out in the log already from the location where the variable is actually defined and used. Because this file doesn't use the variable, it could be incorrect without causing a failure.